install.sh

```diff
@@ -26,11 +29,6 @@ docker pull staltz/ssb-room
 mkdir ~/ssb-room-data
 chown -R 1000:1000 ~/ssb-room-data
 
-#
-# Redirect internal 8007 to external 80
-#
-sudo iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 80 -j REDIRECT --to-port 8007
```

Comment:
Is this removal causing the website (at port 80) to not load?

Reply:
No, Caddy listens on port 80 and 443 and shares network with the host, and is a reverse proxy for `localhost:8007`.